modules/ATOSBase/src/ATOSbase.cpp

```diff
@@ -18,10 +18,10 @@ ATOSBase::ATOSBase()
     : Module(ATOSBase::moduleName),
 	exitSub(*this, std::bind(&ATOSBase::onExitMessage, this, _1))
 {
-	declare_parameter("test_origin_latitude");
-	declare_parameter("test_origin_longitude");
-	declare_parameter("test_origin_altitude");
-	declare_parameter("test_origin_rot");
+	declare_parameter("test_origin_latitude", 57.77744364201314);
+	declare_parameter("test_origin_longitude", 12.7809807263866);
+	declare_parameter("test_origin_altitude",193.114);
+	declare_parameter("test_origin_rot",0.0);
 
```

Comment:
test origin is changed on multiple places - is this another one?

Reply:
This is just a declaration of the parameter for ROS, it needs to have a default value, perhaps something better could be choosen, null island perhaps?

Reply:
Changed it to null island instead

Reply:
not sure if ros will like that on a clean install. 
I think I tried it and it complains with the types, it wanted to know if it was floats int:s etc.... don't know why